src/io_reading_files/FileReading.java

```diff
@@ -0,0 +1,112 @@
+package io_reading_files;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FileReading {
+    public static void main(String[] args) {
+        FileReading fileReading = new FileReading();
+
+        fileReading.sortedNamesbyAgeOlder17("task3.csv");
+
+        Customer customer1 = new Customer("Ivan", "Ivanov", 18);
+        Customer customer2 = new Customer("Petr", "Petrov", 45);
+        Customer customer3 = new Customer("Sidr", "Sidorov", 1);
+        List customers = Arrays.asList(customer1, customer2, customer3);
+        File file = new File("task4.csv");
+        fileReading.makeFileFromCustomersList(customers, file);
+    }
+
+    /* 1.1.Есть текстовый файл, где каждая строка - целое число:1254554Написать функцию,
+    которая принимает имя файла,  возвращает сумму чисел из файла.Подсказка: FileReader, BufferedReader плюс stream API */
+
+    public int readFile(String str) {
+        int sum = 0;
+        try (BufferedReader in = new BufferedReader(new FileReader(str));) {
+            sum = in.lines()
+                    .mapToInt(Integer::valueOf)
+                    .reduce(0, (i1, i2) -> i1 + i2);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        return sum;
+    }
+
+     /* 2. Есть массив строк разной длины. Отсортировать его по длине строк.
+    Пример:вход ​{“aaa”, “bbbb”, “a”, ‘bb”}​, выход ​{“a”, “bb”, “aaa”, “bbbb”}​
+    . Подсказка:Stream API, sorted, Comparator, lambda functions, Java Doc*/
+
+    public List SortedByStringLength(String str) {
+
+        List sortedLines = new ArrayList();
+
+        try (BufferedReader in = new BufferedReader(new FileReader(str))) {
+            sortedLines = in.lines()
+                    .sorted(Comparator.comparingInt(String::length))
+                    .collect(Collectors.toList());
+
+
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        return sortedLines;
```

Comment:
in case the list is not initialized to this point, I would use `return Collections.emptyList();`. May be worth placing such return to the catch section. 
Something like this:
```
try {
 return in.lines()...
} catch() {
 return Collections.emptyList();
}
```